### ISSUE TITLE
Add writableMember struct to RepositoryGroup 

### ISSUE THREAD

**Issue by chewrocca** [datadrivers/go-nexus-client#46]
Looks like with Nexus Pro there is a new option for Writable Repository.

[Pushing Images](https://help.sonatype.com/repomanager3/formats/docker-registry/pushing-images)

/v1/repositories/docker/group/{repositoryName} PUT in API now has this option for writableMember string:

```sh
{
  "name": "internal",
  "online": true,
  "storage": {
    "blobStoreName": "default",
    "strictContentTypeValidation": true
  },
  "group": {
    "memberNames": [
      "string"
    ],
    "writableMember": "string"
  },
  "docker": {
    "v1Enabled": false,
    "forceBasicAuth": true,
    "httpPort": 8082,
    "httpsPort": 8083
  }
}
```

**Comment by anmoel** [comment]
Hi @chewrocca,

thanks for your report.
We will add this requested feature to our list and will implement them in the next few weeks.

regards
André

**Comment by anmoel** [comment]
fixed with release v1.0.0
